### PR TITLE
More understandable message

### PR DESCRIPTION
**Pull request by shapkarin**

### CODE REVIEW

File: src/views/Home.vue [hunk 1]
```diff
@@ -31,7 +31,7 @@ export default {
   data: () => ({ sdk: null }),
   methods: {
     async joinMeeting() {
-      const message = `I would like to generate JWT token at ${new Date().toUTCString()}`;
+      const message = `I would like to login with Superhero ID at ${new Date().toUTCString()}`;
```
Comment (venimus) [inline src/views/Home.vue:34]:
```suggestion
      const message = `Login with Superhero ID at ${new Date().toUTCString()}`;
```

File: srv/index.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ import { readFileSync } from 'fs';
 import { sign } from 'jsonwebtoken';
 import fetch from 'node-fetch';
 
-const messageRegExp = /^I would like to generate JWT token at (?<date>.*)$/;
+const messageRegExp = /^I would like to login with Superhero ID at (?<date>.*)$/;
```
Comment (venimus) [inline srv/index.js:10]:
```suggestion
const messageRegExp = /^Login with Superhero ID at (?<date>.*)$/;
```

**Review by venimus — APPROVED** [review]

**Review by davidyuk — COMMENTED** [review]
What is Superhero ID?

**Comment by shapkarin** [comment]
> What is Superhero ID?

@davidyuk That means `superhero account`. So I suggest to rename `ID` to the `account`.

File: src/views/Home.vue [hunk 3]
```diff
@@ -31,7 +31,7 @@ export default {
   data: () => ({ sdk: null }),
   methods: {
     async joinMeeting() {
-      const message = `I would like to generate JWT token at ${new Date().toUTCString()}`;
+      const message = `Login with Superhero ID at ${new Date().toUTCString()}`;
```
Comment (shapkarin) [inline src/views/Home.vue:34]:
```suggestion
      const message = `Login with Superhero account at ${new Date().toUTCString()}`;
```

File: srv/index.js [hunk 4]
```diff
@@ -7,7 +7,7 @@ import { readFileSync } from 'fs';
 import { sign } from 'jsonwebtoken';
 import fetch from 'node-fetch';
 
-const messageRegExp = /^I would like to generate JWT token at (?<date>.*)$/;
+const messageRegExp = /^Login with Superhero ID at (?<date>.*)$/;
```
Comment (shapkarin) [inline srv/index.js:10]: const messageRegExp = /^Login with Superhero account at (?<date>.*)$/;

**Comment by davidyuk** [comment]
And how `superhero account` connected to message signing? I won't agree that this PR made the message more understandable 🙃

**Comment by shapkarin** [comment]
@davidyuk sure, it's just a suggestion. But I think that `account` is better than `ID`. the main goal is to make messages as much understandable as it can me. We need to find some balance between technical description and understand for non tech users. Lets discuss that, maybe at some issue,

**Pull request merged**